extensions/common/context.cc

```diff
@@ -83,6 +102,19 @@ void populateHTTPRequestInfo(RequestInfo *request_info) {
       getHeaderMapValue(HeaderMapType::RequestHeaders, kMethodHeaderKey)
           ->toString();
   getRequestDestinationPort(&request_info->destination_port);
+
+  std::string tls_version;
+  bool cert_presented;
+
+  if (outbound) {
+    getResponsePeerCertificatePresented(&cert_presented);
```

Comment:
Currently both these are outbound.  We will need Kuat's fixes and then he may change the API as well.